python/ray/air/tests/test_checkpoint_manager.py

```diff
@@ -97,6 +97,40 @@ def test_keep_best_checkpoints():
     ] == [1, 0]
 
 
+def test_nested_get_checkpoint_score():
```

Comment:
Can we also add the nested metric with `/` as a test case?

Reply:
Added!